crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -244,9 +244,10 @@ pub(crate) trait StateWriteExt: StateWrite {
         self.put_account_balance(
             &address,
             asset,
-            balance
-                .checked_sub(amount)
-                .context("subtracting from account balance failed due to insufficient funds")?,
+            balance.checked_sub(amount).context(format!(
+                "subtracting from account balance failed due to insufficient funds; balance: \
+                 {balance}, subtraction amount: {amount}",
+            ))?,
```

Comment:
It looks like this change has nothing to do with the rest of the PR? Can you move it to a new PR?